recipe/meta.yaml

```diff
@@ -10,12 +10,16 @@ source:
   md5: 971eee85d630eb4bafcd52531c79673f
 
 build:
-  number: 0
-  script: python setup.py install
+  number: 1
+  script:
+    - python setup.py install
+    - python -m ipykernel install --prefix $PREFIX
```

Comment:
I did not know you could do a list of lines here. 👍 

Only downside - ```$PREFIX``` isn't going to work with Windows (```%PREFIX%```). I don't really have a workaround for that other than duplicating the line and putting in selectors 😕